### PR TITLE
Update to work with newest protocol

### PR DESCRIPTION
**Pull request by efyang**
Adds dial turn flashing, uses new Led command, and allows for pulse interval setting as well. Waiting on #24.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ cortex-m = "0.7"
 cortex-m-rt = "0.6"
 panic-reset = "0.1"
 nb = "1"
-panel-protocol = { git = "https://github.com/tonarino/panel-protocol.git", rev = "0.4" }
+panel-protocol = { git = "https://github.com/tonarino/panel-protocol.git", branch = "set-pulse-interval" }
```
Comment (efyang) [inline Cargo.toml:15]: This can be reverted once https://github.com/tonarino/panel-protocol/pull/8 is merged.

**Comment by bschwind** [comment]
I see the `Cargo.toml` change in the commits, but not in the overall diff. Has something weird happened with the commit history?

**Comment by efyang** [comment]
The original repo was actually already using "0.4" in the `Cargo.toml` file (even though that didn't exist yet), so the changes in the commits were purely to change only the `Cargo.lock` file and to test the PR while the `panel-protocol` PR hadn't been merged yet.

**Comment by bschwind** [comment]
I see, I think I made some confusion by depending on a 0.4 version even though it didn't exist yet...or something, I'm not quite sure how that went down.

**Review by bschwind — APPROVED** [review]
Tested and everything seems to work, LGTM!

**Pull request merged**